README.md

```diff
@@ -84,4 +84,20 @@ you are with modern services.
 
 Thanks a lot for giving a try. Good luck ;)
 
-Regards, Dom
\ No newline at end of file
+Regards, Dom
+
+## Answer about Description of the Tournament back-end service
+
+1. I highly recommend using socket services to implement the features. we can keep the up-to-date needs data and it will be updated on the client-side whatever its related data is changed.
+
+2. we can choose a very standard communication way between the server and client-side. I mean, if those data will not be needed to update often, we don't need to send the requests to get the latest data to the server by using API. Whenever the user visits this page, we can get the necessary data using API ( fetch or axios or something )
+
+3. we can make the perioded requests by the client-side to get the updated information but we need to make sure that it should be set up as well to let the server not down or very low performance.
+I don't recommend its way however we might choose it according to the specifications.
+
+I am a strong believer that any kind of web service and infrastructure must be depended on the specifications. There are lots of things to be considered between large-scale services and medium or small applications such as worth, cost vs profit, performance, maintenance, etc.
```

Comment:
You have a valid point:
- Completely understanding the task scope is essential before starting any development.
- At a larger scale, it's important to understand the project parameters before selecting a technology stack...

Just for the sake of the discussion:
- If you consider, the JSON data represents tournament data that pretty complete and static
- Once information about the tournament are published, when people start to register, the product team will have to go ahead... It's only near the tournament start that they can pull it off if there not enough registration, for example.
- If the tournament is pulled off, that won't impact the tournament information per-se, right? Only the call telling what are the upcoming tournament will just stop return its identifier...
- As such the tournament information, the JSON payload in the file, can just stay on disk, on AWS S3 specifically! No need of a custom service, and with S3 we only pay for the space and the bandwidth, not for CPU and memory...

Regarding your first suggestion:
- Maintaining a fleet of service to forward information via websocket can be very expensive. It should be reserved to data exchanges that have a high ROI ;)